src/ucp/wireup/select.c

```diff
@@ -1268,6 +1269,10 @@ ucp_wireup_add_rma_bw_lanes(const ucp_wireup_select_params_t *select_params,
     ucp_tl_bitmap_t tl_bitmap;
     uint8_t i;
 
+    if (ep_init_flags & UCP_EP_INIT_CM_PHASE) {
```

Comment:
pls rename UCP_EP_INIT_CM_PHASE to what it does and not how it's used

Reply:
I think this is suitable name, because it indicates that we do EP configuration selection on CM phase

Reply:
per offline discussion, keep as is